internal/observability/event/sink_socket.go

```diff
@@ -29,9 +30,19 @@ type SocketSink struct {
 
 // NewSocketSink should be used to create a new SocketSink.
 // Accepted options: WithMaxDuration and WithSocketType.
-func NewSocketSink(format string, address string, opt ...Option) (*SocketSink, error) {
+func NewSocketSink(address string, format string, opt ...Option) (*SocketSink, error) {
```

Comment:
why choose to switch the order of the arguments? do we need to modify any callers to support the new order?